docs/ref/models/indexes.txt

```diff
@@ -32,6 +32,19 @@ references of :class:`Index` which includes the `index options`_.
 
 A list of the name of the fields on which the index is desired.
 
+By default the indexes are created with an ascending order for all the columns.
+To define an index with a descending order for a particular column, a hyphen
+(-) can be added before the field's name while defining the index.
+
+For example::
+
+    Index(fields=['headline', '-pub_date'], name='headline_pub_date_reverse_idx')
+
+The above index would be issue an SQL with ``(headline, pub_date DESC)``. This
+is supported only for PostgreSQL, Oracle and some versions of SQLite. If your
+database doesn't support defining ordering on index columns it will simply be
```

Comment:
The idea for the test is to provide that specifying a sort order in the SQL is indeed ignored and doesn't crash the database. I'm not sure if some other database might require the SQL to be omitted but such a test would detect that case and alert a maintainer to notify us that we should use the feature flag in django/db/models/indexes.py
 to strip out the DESC SQL.

I'm not sure if the unsupported versions of SQLite are important to consider. Generally the feature flag would be computed based on the database information, e.g. https://github.com/django/django/blob/ac1975b18b5a33234284bec86e5a5bb44a4af976/django/db/backends/mysql/features.py#L51-L54